cypress/integration/page.spec.js

```diff
@@ -1062,9 +1140,15 @@
 
       cy.get('@commentsPost').should('be.calledOnce');
       cy.get('@commentsPostRequest').its('request.body.postId').should('eq', 1);
-      cy.get('@commentsPostRequest').its('request.body.name').should('eq', 'Some name');
-      cy.get('@commentsPostRequest').its('request.body.email').should('eq', 'some@email.com');
-      cy.get('@commentsPostRequest').its('request.body.body').should('eq', 'Some comment body');
+      cy.get('@commentsPostRequest')
+        .its('request.body.name')
+        .should('eq', 'Some name');
+      cy.get('@commentsPostRequest')
+        .its('request.body.email')
+        .should('eq', 'some@email.com');
+      cy.get('@commentsPostRequest')
+        .its('request.body.body')
+        .should('eq', 'Some comment body');
```

Comment:
Ensure that the `cy.get` and `its` statements are properly chained and terminated with semicolons. This ensures proper execution of the assertions.